### PR TITLE
Feature: Add Websocket Metadata Support for Ticket Operations

### PR DESCRIPTION
**Pull request by saithsab877**
### Describe the Changes:
- Add websocket metadata wrapper to ticket creation and update endpoints for real-time sync support

closes: stakwork/sphinx-tribes-frontend#712

## Issue ticket number and link:
- **Ticket Number:** [ 2083 ]
- **Link:** [ https://github.com/stakwork/sphinx-tribes-frontend/issues/712 ]

### Evidence:

image

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome
- [x] New feature (non-breaking change which adds functionality)
- [x] I have provided a screenshot of changes in my PR

### CODE REVIEW

**Comment by saithsab877** [comment]
Hi @humansinstitute, Please review this PR.

**Comment by saithsab877** [comment]
Hi @humansinstitute, Please review this PR.

File: src/people/widgetViews/PhasePlannerView.tsx [hunk 1]
```diff
@@ -147,8 +147,16 @@ const PhasePlannerView: React.FC = () => {
       version: 1
     };
 
+    const ticketPayload = {
+      metadata: {
+        source: 'websocket',
+        id: initialTicketData.uuid
```
Comment (humansinstitute) [inline src/people/widgetViews/PhasePlannerView.tsx:153]: This isn't correct. 
We need metadata.id = the websocket session ID

This is setup on page load of phasePlanner and logged out in the code if you check the ticket.

**Comment by saithsab877** [comment]
Hi @humansinstitute, Please review this PR.

**Pull request merged**